src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -362,12 +423,28 @@ also be specified.
 
 General fields:
 
+{{ MACROS___make_metadata_table(
+   {
+      "IntendedFor": "OPTIONAL",
+   }
+) }}
+
 | **Key name** | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | ------------ | --------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | IntendedFor  | RECOMMENDED           | [string][]    | This can be an MRI/CT or a file containing the operative photo, x-ray or drawing with path relative to the project folder. If only a surface reconstruction is available, this should point to the surface reconstruction file. Note that this file should have the same coordinate system specified in `iEEGCoordinateSystem`. For example, **T1**: `"sub-<label>/ses-<label>/anat/sub-01_T1w.nii.gz"`  **Surface**: `"/derivatives/surfaces/sub-<label>/ses-<label>/anat/sub-01_T1w_pial.R.surf.gii"` **Operative photo**: `"/sub-<label>/ses-<label>/ieeg/sub-0001_ses-01_acq-photo1_photo.jpg"` **Talairach**: `"/derivatives/surfaces/sub-Talairach/ses-01/anat/sub-Talairach_T1w_pial.R.surf.gii"` |
 
 Fields relating to the iEEG electrode positions:
 
+{{ MACROS___make_metadata_table(
+   {
+      "iEEGCoordinateSystem": "REQUIRED",
+      "iEEGCoordinateUnits": "REQUIRED",
+      "iEEGCoordinateSystemDescription": 'RECOMMENDED, but REQUIRED if `iEEGCoordinateSystem` is `"Other"`',
```

Comment:
oh, that is an interesting case...  in light of my eager desire for concentrating this information in machine readable form, I can suggest for such cases to get to temp middle point  where requirement could be "spelled out" in human form for now with the last one listed as "catch all", e.g. 

```yaml
metadata:
 iEEGCoordinateUnits: required
 iEEGCoordinateUnits: required
 iEEGCoordinateSystemDescription:
  required: if `iEEGCoordinateSystem` is `"Other"`
  recommended:
 iEEGCoordinateProcessingDescription: recommended
```